src/compas_view2/views/view.py

```diff
@@ -155,6 +155,10 @@ def resize(self, w, h):
         pass
 
     def paintGL(self):
+        if self.current != self.PERSPECTIVE:
+            self.shader.bind()
+            self.shader.uniform4x4("projection", self.camera.projection(self.app.width, self.app.height))
+            self.shader.release()
```

Comment:
shouldn't this move to the `paint` method of `View120`, since it is shader specific?
also, somewhat related, is there a difference between `selector.enabled` and `selector.pain_instance`?

Reply:
i guess other stuff would have to move as well then, because the instance map would not be correct...

Reply:
but by moving all shader specific stuff to the shader version of the view class, things will be a bit more transparent

Reply:
Exactly, the projection update has to be before all types of paints. But you are right about that we should move all these to view120. let me update the branch
the selector.enabled is redundant currently, will clean that up a bit too.

Reply:
well but can we not switch to `selector.enabled` instead of `selector.paint_instance`? i find the former more meaningful than the latter...

Reply:
sure gonna delete paint_instance and only use enabled instead